pkg/crc/preflight/preflight_checks_common.go

```diff
@@ -154,15 +155,13 @@ func removeCRCMachinesDir() error {
 }
 
 func removeOldLogs() error {
-	logFiles, err := filepath.Glob(filepath.Join(constants.CrcBaseDir, "*.log_*"))
-	if err != nil {
-		return fmt.Errorf("Failed to get old logs: %w", err)
+	// remove crc cli old log files
+	if err := crcos.RemoveFileGlob(filepath.Join(constants.CrcBaseDir, "crc-*.log")); err != nil {
+		return fmt.Errorf("Failed to remove old log files: %w", err)
 	}
-	for _, f := range logFiles {
-		logging.Debugf("Deleting %s log file", f)
-		if err := os.RemoveAll(f); err != nil {
-			return fmt.Errorf("Failed to delete %s: %w", f, err)
-		}
+	// remove daemon old log files
+	if err := crcos.RemoveFileGlob(filepath.Join(constants.CrcBaseDir, "crcd-*.log")); err != nil {
+		return fmt.Errorf("Failed to remove old log files: %w", err)
```

Comment:
Shouldn't `err := crcos.RemoveFileGlob(filepath.Join(constants.CrcBaseDir, "*.log"))` be enough for delete `crcd-` and `crc-` log files.

Reply:
with this we want to only delete the "old" log files, that were backed up, and not delete the `crc.log` and `crcd.log` files

the backup log files are named like `crc-xxxx-xxx-xxx.log` and `crcd-xxxx-xxxx-xxxx.log`, we can combine the two though with just `crc-*.log`

Reply:
@anjannath yes but my point is when we are doing `crc cleanup` then shouldn't we also delete `crc.log` and `crcd.log` or only want to delete the backups?

Reply:
well this was the existing behavior, but yeah i think we can delete these two files as well, if a user is running `cleanup` most probably next they will start a new instance

Reply:
@anjannath if the cleanup process also need logging then in the windows cleanup might fail because `crc.log` and `crcd.log` file might still be in use. Let's test this PR in the windows to make sure it doesn't break.

Reply:
hmm, will test on windows, might be one of the reason why only backup log files were removed and not the `crc.log` and `crcd.log` files, `crcd.log` is still a bit safer as the daemon will be killed before the log file removal i think

Reply:
@praveenkumar So tested on windows and you were right, it does throw an error "unable to delete those files, as it is used by another process..." i think we have to revert to only deleting the backed up log files, or somehow make this cleanup action run at the very end of the cleanup

Reply:
@praveenkumar reverted to removing only the backup log files, to prevent throwing the "file in use" error on windows